From b0966f17f701d719c673fdd566092ca49b68e934 Mon Sep 17 00:00:00 2001
From: Jeff LaBundy <jeff@labundy.com>
Date: Tue, 29 Jan 2019 19:35:45 -0600
Subject: [PATCH] configs: Enable Azoteq IQS550/572/525

This patch enables the Azoteq IQS550/572/525 trackpad/touchscreen
controller as a module.

Signed-off-by: Jeff LaBundy <jeff@labundy.com>
---
 arch/arm/configs/bcm2709_defconfig   | 1 +
 arch/arm/configs/bcm2711_defconfig   | 1 +
 arch/arm/configs/bcmrpi_defconfig    | 1 +
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcmrpi3_defconfig | 1 +
 5 files changed, 5 insertions(+)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index e844c6d60011..902e7572e863 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -626,6 +626,7 @@ CONFIG_TOUCHSCREEN_EDT_FT5X06=m
 CONFIG_TOUCHSCREEN_RASPBERRYPI_FW=m
 CONFIG_TOUCHSCREEN_USB_COMPOSITE=m
 CONFIG_TOUCHSCREEN_STMPE=m
+CONFIG_TOUCHSCREEN_IQS5XX=m
 CONFIG_INPUT_MISC=y
 CONFIG_INPUT_AD714X=m
 CONFIG_INPUT_ATI_REMOTE2=m
diff --git a/arch/arm/configs/bcm2711_defconfig b/arch/arm/configs/bcm2711_defconfig
index edcc13e4906e..29dae5a427e8 100644
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -639,6 +639,7 @@ CONFIG_TOUCHSCREEN_EDT_FT5X06=m
 CONFIG_TOUCHSCREEN_RASPBERRYPI_FW=m
 CONFIG_TOUCHSCREEN_USB_COMPOSITE=m
 CONFIG_TOUCHSCREEN_STMPE=m
+CONFIG_TOUCHSCREEN_IQS5XX=m
 CONFIG_INPUT_MISC=y
 CONFIG_INPUT_AD714X=m
 CONFIG_INPUT_ATI_REMOTE2=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 95a11f27dfad..e5363f9e7194 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -620,6 +620,7 @@ CONFIG_TOUCHSCREEN_EDT_FT5X06=m
 CONFIG_TOUCHSCREEN_RASPBERRYPI_FW=m
 CONFIG_TOUCHSCREEN_USB_COMPOSITE=m
 CONFIG_TOUCHSCREEN_STMPE=m
+CONFIG_TOUCHSCREEN_IQS5XX=m
 CONFIG_INPUT_MISC=y
 CONFIG_INPUT_AD714X=m
 CONFIG_INPUT_ATI_REMOTE2=m
diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index 62c8654fb68b..7420ccb807f2 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -638,6 +638,7 @@ CONFIG_TOUCHSCREEN_EDT_FT5X06=m
 CONFIG_TOUCHSCREEN_RASPBERRYPI_FW=m
 CONFIG_TOUCHSCREEN_USB_COMPOSITE=m
 CONFIG_TOUCHSCREEN_STMPE=m
+CONFIG_TOUCHSCREEN_IQS5XX=m
 CONFIG_INPUT_MISC=y
 CONFIG_INPUT_AD714X=m
 CONFIG_INPUT_ATI_REMOTE2=m
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index c4dff0b42461..c7d9a50ec713 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -613,6 +613,7 @@ CONFIG_TOUCHSCREEN_EKTF2127=m
 CONFIG_TOUCHSCREEN_RASPBERRYPI_FW=m
 CONFIG_TOUCHSCREEN_USB_COMPOSITE=m
 CONFIG_TOUCHSCREEN_STMPE=m
+CONFIG_TOUCHSCREEN_IQS5XX=m
 CONFIG_INPUT_MISC=y
 CONFIG_INPUT_AD714X=m
 CONFIG_INPUT_ATI_REMOTE2=m
-- 
2.32.0

